src/wakatime.rs

```diff
@@ -83,10 +72,15 @@ impl WakatimeExtension {
             zed::download_file(
                 &asset.download_url,
                 &version_dir,
-                zed::DownloadedFileType::GzipTar,
+                match platform {
+                    zed::Os::Mac | zed::Os::Linux => zed::DownloadedFileType::GzipTar,
+                    zed::Os::Windows => zed::DownloadedFileType::Zip,
+                },
             )
             .map_err(|e| format!("failed to download file: {e}"))?;
 
+            zed::make_file_executable(&binary_path).expect("failed to make file executable");
```

Comment:
I'm wondering if panicking is better than returning an error? idk how zed handles this in its UI but returning an error seems more idiomatic

but it's probably fine tbh

Reply:
I don't particularly appreciate panicking since in rare cases is it the correct way to handle errors

Reply:
yeah, but [`Result.expect` causes a panic too](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect), it just adds an error message

its fine for this PR like i said, it's always sth that can be fixed later on (extension debugging in zed is pretty lackluster anyway, especially for things that happend before the lsp connection starts)

Reply:
Okay

Reply:
I'm merging then! thx a lot ^^